applications/nrf5340_audio/README.rst

```diff
@@ -1273,6 +1279,8 @@ Complete the following steps to test the unidirectional CIS mode for one gateway
 #. Press the **BTN 4** button on the gateway multiple times.
    For each button press, the audio stream playback is stopped and the gateway sends a test tone to both headsets.
    These tones can be used as audio cues to check the synchronization of the headsets.
+#. Press the **RESET** button on the gateway and hold **VOL+** on one of the headsets.
+   After startup, the playback stream will be changed to the headset to the right of the headset on which you pressed the volume button.
```

Comment:
```suggestion
   After startup, this headset will now be configured as a right channel headset. You can change it back to left by doing the same step, but using the **VOL-** button instead of the **VOL+** button.
```